README.md

```diff
@@ -194,6 +194,52 @@ In other words, the new flag would indicate several connected changes related to
 
 For more discussion see [issue 2](https://github.com/tc39/proposal-regexp-set-notation/issues/2).
 
+### How is the `v` flag different from the `u` flag?
+
+The answer to this question can be useful when “upgrading” existing `u` RegExps to use `v`. Here’s an overview of the differences:
+
+1. (This is the obvious part.) Previously invalid patterns making use of the new syntax (see above) now become valid, e.g.
+
+    ```
+    [\p{ASCII_Hex_Digit}--[Ff]]
+    \p{RGI_Emoji}
+    [_\q{a|bc|def}]
+    ```
+
+1. Some previously valid patterns are now errors, specifically those with a character class including either an unescaped [special character](https://arai-a.github.io/ecma262-compare/snapshot.html?pr=2418#prod-ClassSetSyntaxCharacter) `(` `)` `[` `{` `}` `/` `-` `|` (note: `\` and `]` also require escaping inside a character class, but this is already true with the `u` flag) or [a double punctuator](https://arai-a.github.io/ecma262-compare/snapshot.html?pr=2418#prod-ClassSetReservedDoublePunctuator):
+
+    ```
+    [(]
+    [)]
+    [[]
+    [{]
+    [}]
+    [/]
+    [-]
+    [|]
+    [&&]
+    [!!]
+    [##]
+    [$$]
+    [%%]
+    [**]
+    [++]
+    [,,]
+    [..]
+    [::]
+    [;;]
+    [<<]
+    [==]
+    [>>]
+    [??]
+    [@@]
+    [^^]
```

Comment:
@gibson042 @markusicu @macchiati @pthier Is `/[^^]/v` actually an error? I noticed the current V8 implementation allows it.

Reply:
IMO it isn't, as the first `^` is consumed by the production `CharacterClass :: [ ^ ClassContents ]`, so the remaining `^` is not a reserved double punctuator. `[^^^]` is an error in V8.
Note that also `[]]` is not a breaking change, as this is an error also with `/u`.

Reply:
I guess `[_^^]` or, indeed, `[^^^]` is a better example. I’ll change this. Thanks!

`[]]` is currently not listed as a breaking change, so I think there’s no action to be taken for that one. Let me know if I misunderstood.

Reply:
PR: https://github.com/tc39/proposal-regexp-v-flag/pull/74